<!-- -->
<launch>
	<arg name="save_finename" default="/media/pdsljp/ssd/test.txt" />
	<arg name="save_topic" default="ndt" /> <!--  amcl or ndt or gnss -->
<arg name="interval" default="1" /> 
<arg name="no_velocity" default="false" />

<!-- rosrun waypoint_maker waypoint_saver _save_topic:=[topic] _interval:=[value] _save_filename:="[save file]" -->
	<node pkg="waypoint_maker" type="waypoint_saver" name="waypoint_saver" output="log">
		<param name="save_filename" value="$(arg save_finename)" />
		<param name="save_topic" value="$(arg save_topic)" />
		<param name="interval" value="$(arg interval)" />
		<param name="no_velocity" value="$(arg no_velocity)" />
	</node>

</launch>
